.github/workflows/network-deploy.yml

```diff
@@ -38,12 +38,12 @@ jobs:
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: us-east-2
```

Comment:
What's the rationale here?

Reply:
The original cluster was in us-east-2. Since I was using the same "spartan" naming, I deployed in the next closest region to make it easy for me to go and clean up the old cluster resources without confusing any of resource names. 

I didn't want to delete the old resources before the new ones were online, because the originals were not deployed using IaC and couldn't be restored if there was an issue.

Reply:
There were other resources, such as the VPC which sat outside of k8s as well, so mostly this was ease of cleanup for the old items.